### PR TITLE
docs: enable tooltips

### PR DESCRIPTION
**Pull request by dgarcia360**
Related issue #873

Updates the docs theme to the [latest version](https://sphinx-theme.scylladb.com/stable/upgrade/CHANGELOG.html#changelog) to enable tooltips and modifies the `db_options.tmpl` to show the new role in action.

## How to test

1. Clone this PR.
2. Build the docs.
3. Open http://127.0.0.1:5500/reference/configuration-parameters/ and preview the tooltips:

    image

### CODE REVIEW

**Comment by tzach** [comment]
I love the tooltip feature, but in this case (Liveness) the tip is too long, and hard to read.
Maybe less text, bigger font, will help

File: docs/poetry.lock [hunk 1]
```diff
@@ -0,0 +1,1374 @@
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+
```
Comment (avikivity) [inline docs/poetry.lock:2]: Is this part of the commit or accidental?

Comment (dgarcia360) [reply docs/poetry.lock:2]: It's part of the commit (update to the latest version). We now include `poetry.lock` in all projects ([context](https://github.com/scylladb/sphinx-scylladb-theme/issues/849)).

Unless contributors manually run `make update`, the `poetry.lock` file will remain static and won’t be changed in other pull requests.

Comment (denesb) [reply docs/poetry.lock:2]: In the future, consider doing this as a separate PR or at least as a separate commit, so it doesn't get lost in the noise.

**Pull request closed without merging**